modules/phase_field/doc/content/source/ics/SmoothCircleIC.md

```diff
@@ -1,15 +1,34 @@
-<!-- MOOSE Documentation Stub: Remove this when content is added. -->
-
 # SmoothCircleIC
 
-!alert construction title=Undocumented Class
-The SmoothCircleIC has not been documented, if you would like to contribute to MOOSE by
-writing documentation, please see [/generate.md]. The content contained on this page explains
-the typical documentation associated with a MooseObject; however, what is contained is ultimately
-determined by what is necessary to make the documentation clear for users.
-
 !syntax description /ICs/SmoothCircleIC
 
+Insert a single disk or sphere with radius $R$ (`radius`) and a smooth interface
+of width $w_i$ (`int_width`) with a user specified value range $[v_0, v_1]$ and
+functional dependence of the interface (`profile`). $v_0$ (`outvalue`) is the
+variable value outside of the disk/sphere and $v_1$ (`invalue`) is the value on
+the interior.
+
+### Cosine profile (`COS`)
+
+\begin{equation}
+v(\vec r) = x\begin{cases}
+        v_1, & \text{for } |\vec r| \leq R - \frac w2\\
+        v_0 +  \frac12(v_1 - v_0) \left(1.0 + \cos(\pi\frac{|\vec r| - R\ + \frac w2}w)\right)
+, & \text{for } R - \frac w2\lt |\vec r| \lt R + \frac w2\\
+        v_0, & \text{for } R + \frac w2 \leq |\vec r|
+        \end{cases}
+\end{equation}
+
+### Hyperbolic tangent profile (`TANH`)
+
+\begin{equation}
+v(\vec r) = (v_1 - v_0) \frac12 \left(\tanh(4 \frac{R - |\vec r|}w ) + 1\right) + v_0;
```

Comment:
Why the factor of 4? Is that just to make the actual width closer to what's specified in the parameter `w`? I think the reason for it is worth mentioning.

Reply:
You comment made me think of a more rigorous definition of "equal interface width" I have changed the 4 to pi and added an explanation and a plot in the docs.